### ISSUE TITLE
WCAG Translations check links

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-translations#113]
https://validator.w3.org/checklink?url=https://www.w3.org/WAI/standards-guidelines/wcag/translations/

**Comment by remibetin** [comment]
Results:

### This is a permanent redirect. The link should be updated to point to the more recent URI.

⚠️  Lines: 185, 523 http://www.w3.org/Consortium/Translation/ redirected to https://www.w3.org/Consortium/Translation/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated to point to the more recent URI.
Broken fragments:
http://www.w3.org/Consortium/Translation/#auth (line 185)

### This is a server side problem. Check the URI.

error Line: 472 http://www.acesso.umic.pt/w3/TR/WCAG20-quickref/
Status: 500 Can't connect to www.acesso.umic.pt:80 (Name or service not known)
This is a server side problem. Check the URI.

error Line: 471 http://www.acesso.umic.pt/w3/TR/WCAG20-TECHS/
Status: 500 Can't connect to www.acesso.umic.pt:80 (Name or service not known)
This is a server side problem. Check the URI.

error Line: 457 http://fdc.org.pl/wcag2/
Status: 500 Can't connect to fdc.org.pl:80 (Name or service not known)
This is a server side problem. Check the URI.

error Line: 437 http://archi.ssu.ac.kr/WCAG20/
Status: 500 Can't connect to archi.ssu.ac.kr:80 (Name or service not known)
This is a server side problem. Check the URI.

error Line: 470 http://www.acesso.umic.pt/w3/TR/UNDERSTANDING-WCAG20/
Status: 500 Can't connect to www.acesso.umic.pt:80 (Name or service not known)
This is a server side problem. Check the URI.

### List of redirects

warning Line: 452 http://www.w3.org/Translations/WCAG20-no/ redirected to https://www.w3.org/Translations/WCAG20-no/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 459 http://lists.w3.org/Archives/Public/w3c-translators/2010JanMar/0100.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2010JanMar/0100.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 427 http://www.w3.org/Translations/WCAG20-it/ redirected to https://www.w3.org/Translations/WCAG20-it/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 360 http://www.w3.org/translations/wcag20-zh/ redirected to https://www.w3.org/Translations/WCAG20-zh/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 380 http://www.w3.org/Translations/WCAG20-nl/ redirected to https://www.w3.org/Translations/WCAG20-nl/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 418 http://lists.w3.org/Archives/Public/w3c-translators/2009AprJun/0147.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2009AprJun/0147.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 443 http://lists.w3.org/Archives/Public/w3c-translators/2011JulSep/0063.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2011JulSep/0063.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 401 http://lists.w3.org/Archives/Public/w3c-translators/2010JanMar/0013.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2010JanMar/0013.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 433 http://lists.w3.org/Archives/Public/w3c-translators/2009JanMar/0198.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2009JanMar/0198.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 478 http://lists.w3.org/Archives/Public/w3c-translators/2010JanMar/0031.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2010JanMar/0031.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 412 http://www.isoc.org.il/w3c-wai/guidelines.html redirected to https://www.isoc.org.il/files/w3c-wai/guidelines.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 390 http://www.w3.org/Translations/WCAG20-fi/ redirected to https://www.w3.org/Translations/WCAG20-fi/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 547 http://www.w3.org/People/Shawn/ redirected to https://www.w3.org/People/Shawn/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 375 http://www.w3.org/Translations/WCAG20-da/ redirected to https://www.w3.org/Translations/WCAG20-da/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 400 http://www.w3.org/Translations/WCAG20-de/ redirected to https://www.w3.org/Translations/WCAG20-de/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 482 http://www.w3.org/Translations/WCAG20-ru/ redirected to https://www.w3.org/Translations/WCAG20-ru/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 396 http://www.w3.org/Translations/NOTE-UNDERSTANDING-WCAG20-fr/ redirected to https://www.w3.org/Translations/NOTE-UNDERSTANDING-WCAG20-fr/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 385 http://www.w3.org/Translations/WCAG20-et/ redirected to https://www.w3.org/Translations/WCAG20-et/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 184 http://www.w3.org/2005/02/TranslationPolicy redirected to https://www.w3.org/2005/02/TranslationPolicy
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 490 http://lists.w3.org/Archives/Public/w3c-translators/2009OctDec/0280.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2009OctDec/0280.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 346 http://lists.w3.org/Archives/Public/w3c-translators/2010JulSep/0113.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2010JulSep/0113.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 497 http://www.w3.org/Translations/WCAG20-sv/ redirected to https://www.w3.org/Translations/WCAG20-sv/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 350 http://www.fatcow.com/resources/wai-be.html redirected to https://www.bluehost.com/fatcow?brand=fatcow&siteID=46097236&channelID=P61C46097236S653N0B2A151D0E0000V100
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 492 http://lists.w3.org/Archives/Public/w3c-translators/2012OctDec/0019.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2012OctDec/0019.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 395 http://www.w3.org/Translations/WCAG20-fr/ redirected to https://www.w3.org/Translations/WCAG20-fr/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 355 http://www.w3.org/Translations/WCAG20-ca/ redirected to https://www.w3.org/Translations/WCAG20-ca/
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 366 http://lists.w3.org/Archives/Public/w3c-translators/2009OctDec/0188.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2009OctDec/0188.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 438 http://lists.w3.org/Archives/Public/w3c-translators/2009JanMar/0077.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2009JanMar/0077.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.

warning Line: 371 http://lists.w3.org/Archives/Public/w3c-translators/2010OctDec/0022.html redirected to https://lists.w3.org/Archives/Public/w3c-translators/2010OctDec/0022.html
Status: 301 -> 200 OK
This is a permanent redirect. The link should be updated.
